walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -27,66 +27,74 @@
         Collection<TimeRange> avaliableTimes = new ArrayList<>();
         long duration = request.getDuration();
         Collection<String> attendees = request.getAttendees();
+        Collection<String> optionalAttendees = request.getOptionalAttendees();
```

Comment:
Not that I suggest you change all your code, but take a look at[ java optionals](https://www.baeldung.com/java-optional);